lib/rules/indent.js

```diff
@@ -608,9 +608,13 @@ module.exports = {
                 return elem !== null;
             });
 
-            // Skip if first element is in same line with this node
-            if (elements.length > 0 && elements[0].loc.start.line === node.loc.start.line) {
-                return;
+            // Skip if element is in same line with this node
+            if (elements.length > 0 && node.type === "ArrayExpression") {
+                elements.forEach(el => {
+                    if (el.loc.start.line === node.loc.start.line) {
+                        return;
+                    }
+                });
```

Comment:
It looks like it was supposed to do this:

```js
elements = elements.filter(el => el.loc.start.line === node.loc.start.line);
```

But I don't think this check is necessary anyway, because `report` will not report a node unless it's the first on its line.

So I think we can just remove this entire block in https://github.com/eslint/eslint/pull/7441.